src/verifier/core/reporting.py

```diff
@@ -107,7 +106,7 @@ def create(self, aid: str, dig: str, lei: str, filename: str, typ: str, stream):
         diger = DigerBuilder.sha256(dig)
         report = b''
         while True:
-            chunk = stream.read(4096)
+            chunk = stream.read(4194304)
```

Comment:
Do you want to do os.getenv("FILER_CHUNK_SIZE") ?

Reply:
Sure, I'll do this way. And I'll make default value to be 4194304(4MB)